src/VisualStudio/Core/Def/ExternalAccess/VSTypeScript/Api/IVsTypeScriptRemoteLanguageServiceWorkspace.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.VisualStudio.LanguageServices.ExternalAccess.VSTypeScript.Api
+{
+    /// <summary>
+    /// Used to acquire the RemoteLanguageServiceWorkspace. Its members should be accessed by casting to <see cref="Workspace"/>.
+    /// </summary>
+    interface IVsTypeScriptRemoteLanguageServiceWorkspace
```

Comment:
should this just implement workspace?

Reply:
Switched to abstract class that extends `Workspace`.